test/cmd/certs.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+os::test::junit::declare_suite_start "cmd/admin/certs-validation"
+
+CERT_DIR="${BASETMPDIR:-/tmp}/certs"
+mkdir -p "${CERT_DIR}"
```

Comment:
@stevekuznetsov Could you take a look on these tests? Thank you in advance!

- I'm creating temp directory inside of `$BASETMPDIR`: is it a right location?
- I'm creating it but I'm not sure whether I should remove it or not.
- I've just create file in the `test/cmd` directory. Is it enough for being picked up by integration tests?